dashboard/serializers.py

```diff
@@ -25,6 +25,25 @@ def serialize_enrollments(cls, enrollments):
         return [{'title': course.title} for course in enrolled_courses]
 
     @classmethod
+    def count_complete_courses(cls, program, mmtrack):
+        """
+        Returns number of complete courses.
+
+        Args:
+            program (Program): A program object.
+            mmtrack (MMTrack): mmtrack utility.
+        Returns:
+            num_complete_courses: A number of complete courses.
+        """
+        num_complete_courses = 0
+        for course in program.course_set.all():
+            course_run = course.first_unexpired_run()
```

Comment:
@giocalitri @noisecapella this code is correct for getting edX course id.

I want to get number of courses complete here